phd-tests/framework/src/guest_os/windows_server_2022.rs

```diff
@@ -0,0 +1,67 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Guest OS adaptations for Windows Server 2022 images. This adapter assumes
+//! the guest OS image has been generalized, that Cygwin has been installed, and
+//! that the local administrator user was configured at image generation time to
+//! have the appropriate password.
```

Comment:
not at all important, nitpicky: would it make sense for this to be a doc comment on `WindowsServer2022` rather than at the module level?